File: example/scalalib/testing/6-test-group-stealing/foo/test/src/GroupY7.scala
package foo
import utest._
object GroupY7 extends RandomTestsUtils {
  def tests = Tests {
    test("test1") { testGreeting("Horus", 34) }
    test("test2") { testGreeting("Isis", 32) }
    test("test3") { testGreeting("Juno", 33) }
  }
} 

File: example/kotlinlib/testing/6-test-group-stealing/foo/test/src/foo/GroupY7.kt
package foo

import org.junit.jupiter.api.Test

class GroupY7 : RandomTestsUtils() {
    @Test
    @Throws(Exception::class)
    fun test1() {
        testGreeting("Horus", 34)
    }

    @Test
    @Throws(Exception::class)
    fun test2() {
        testGreeting("Isis", 32)
    }

    @Test
    @Throws(Exception::class)
    fun test3() {
        testGreeting("Juno", 33)
    }
}
